docs/indicator-configuration.md

```diff
@@ -0,0 +1,470 @@
+```
+
+### copyright
+
+**_Optional_**: This setting can be used to display copyright information below the graph and table on the indicator page.
+
+```nohighlight
+copyright: Copyright 2019 - My organisation
+```
+
+With translation key:
+
+```nohighlight
+copyright: my_translations.copyright_message
+```
+
+### data_footnote
+
+**_Optional_**: This setting can be used to display additional inforamtion about the data below the graph and table on the indicator page.
+
+
+```nohighlight
+data_footnote: My additional information
+```
+
+With translation key:
+
+```nohighlight
+data_footnote: my_translations.1-1-1-footnote
+```
+
+
+### data_not_statistical
+
+**_Required_**: This setting is used to specify whether the indicator is statistical (can be charted/graphed) or not. If you have uploaded data that you want to display on a chart/table for an indicator, set this to `false`.
+
+```nohighlight
+data_non_statistical: false
+```
+
+You may want to display some very important information which site viewers must keep in mind when using the data provided. To display a notice above the graph in a coloured box, you can use the following fields within the metadata file.
+
+### data_notice_class
+
+**_Optional_**: This setting can be used to set the colour of a data notice using CSS classes.
+
+```nohighlight
+standalone: success
+```
+
+Options out-of-the-box are:
+* success (green)
+* warning (amber)
+* danger (red)
+
+### data_notice_heading
+
+**_Optional_**: This setting can be used to set the title of a data notice.
+
+```nohighlight
+data_notice_heading: Important Note
+```
+
+With translation key:
+
+```nohighlight
+data_notice_heading: my_translations.important_note
+```
+
+### data_notice_text
+
+**_Optional_**: This setting can be used to set the text to display within a data notice.
+
+```nohighlight
+data_notice_text: My note text
+```
+
+With translation key:
+
+```nohighlight
+data_notice_text: my_translations.1-1-1-data-notice
+```
+
+### data_show_map
+
+**_Optional_** This setting can be used to specify whether or not to display a map for the indicator, if you have configured maps. See the [Maps guidance](maps.md) for more information on how to configure maps.
+
+```nohighlight
+data_show_map: true
+```
+
+### data_start_values
+
+**_Optional_**: This setting is used to specify if you would like an indicator load with certain disaggregation values already selected. Using the example below, the indicator would load with both "Females" and "15 to 19" selected.
+
+```nohighlight
+data_start_values:
+  - field: Sex
+    value: Females
+  - field: Age
+    value: 15 to 19
+```
+
+You may want to add an additional feature which isn't created from data, such as an iframe. You can create an extra tab to display this feature by adding the following fields to the metadata file. You can either specify a URL (use `embedded_feature_url`) or some HTML (use `embedded_feature_html`) for the feature you want to embed.
```

Comment:
Similar to data notices, maybe this needs a heading above it like `### Embedded feature settings`?